### PR TITLE
add(main): new package glslang

### PR DESCRIPTION
**Pull request by Maxython**

### CODE REVIEW

File: packages/glslang/build.sh [hunk 1]
```diff
@@ -0,0 +1,31 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/KhronosGroup/glslang
+TERMUX_PKG_DESCRIPTION="OpenGL and OpenGL ES shader front end and validator"
+TERMUX_PKG_LICENSE="BSD"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION=11.13.0
+TERMUX_PKG_SRCURL=https://github.com/KhronosGroup/glslang/archive/$TERMUX_PKG_VERSION.tar.gz
+TERMUX_PKG_SHA256=592c98aeb03b3e81597ddaf83633c4e63068d14b18a766fd11033bad73127162
+TERMUX_PKG_DEPENDS="python"
```
Comment (xtkoba) [inline packages/glslang/build.sh:8]: Is `python` needed?

Comment (xtkoba) [inline packages/glslang/build.sh:8]: Add `libc++`.

File: packages/glslang/build.sh [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+	termux_setup_cmake
+}
+
+termux_step_make() {
+	cmake ${TERMUX_PKG_EXTRA_CONFIGURE_ARGS}
+	ninja -Cbuild-shared
+}
+
+termux_step_make_install() {
+	ninja -C build-shared install
```
Comment (xtkoba) [inline packages/glslang/build.sh:29]: Non-standard. Doesn't work by default?

File: packages/glslang/build.sh [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+	install -Dm644 LICENSE.txt $TERMUX_PREFIX/share/licenses/glslang/LICENSE
```
Comment (xtkoba) [inline packages/glslang/build.sh:30]: Not needed.

File: packages/glslang/build.sh (hunk 1 above)
Comment (xtkoba) [reply packages/glslang/build.sh:8]: Specified as *build* dependency. I don't believe it is needed on run-time.

Comment (xtkoba) [reply packages/glslang/build.sh:8]: BTW, from time to time I see PRs in which `ndk-sysroot` is specified as dependency, which is incorrect.

File: packages/glslang/build.sh (hunk 3 above)
Comment (Maxython) [reply packages/glslang/build.sh:30]: Install license?

File: packages/glslang/build.sh (hunk 2 above)
Comment (Maxython) [reply packages/glslang/build.sh]: Only cmake starts automatically, everything else needs to be specified.

File: packages/glslang/build.sh (hunk 3 above)
Comment (xtkoba) [reply packages/glslang/build.sh:31]: Exactly. Handled by our scripts by default.

File: packages/glslang/build.sh (hunk 2 above)
Comment (xtkoba) [reply packages/glslang/build.sh]: Uh huh, you mean `-Cbuild-shared` cannot be passed to `ninja` without this customization, right?

File: packages/glslang/build.sh (hunk 3 above)
Comment (xtkoba) [reply packages/glslang/build.sh:31]: Note also that `$TERMUX_PREFIX/share/licenses` is nonstandard in Termux to the point that some maintainer tries to `rm -rf` this directory.

File: packages/glslang/build.sh (hunk 1 above)
Comment (Maxython) [reply packages/glslang/build.sh]: Okay, python is not needed to compile the package, but then I will indicate it in the recommendation

Comment (xtkoba) [reply packages/glslang/build.sh]: Not even needed in the recommendation.

Comment (Maxython) [reply packages/glslang/build.sh]: Very high risks that problems with glslang may occur.

**Pull request merged**